tox.ini

```
@@ -57,6 +57,7 @@ deps =
     oldestdeps: zeep<3.5.0
     oldestdeps: h5netcdf<0.9
     oldestdeps: cdflib<0.3.20
+    oldestdeps: pytest<5.5
```

This works for me locally. We just need `pytest-xdist` to be installed before `sunpy` is so it doesn't install later and upgrade `pytest`. Therefore, we don't actually need to specify a minimum version here.
```suggestion
    oldestdeps: pytest<5.5
    oldestdeps: pytest-xdist
```